src/interface.js

```diff
@@ -0,0 +1,89 @@
+$(document).ready(function() {
+  var game = new Game();
+  var calculator = new GameCalculator();
+
+  createScoreCardAndButtonsHTML();
+
+  $('#score').click(function(){
+    var gameArr = $('#full-game').val().replace(/\s/,'').split(',').map(Number);
+    $('#full-game').val('');
+    $('#final-score').text('Total score: ' + calculator.score(gameArr));
+  });
+
+  $('#restart').click(function(){
+    location.reload();
+  });
+
+  $('.roll-score').click(function(){
+    game.roll = parseInt(this.value);
+    game.addRollToFrame();
+    game.frameCount < MAX_FRAMES - 1 ? displayFrames() : displayFrameTen();
+    game.checkFramesAndRemainingPins();
+    hideOrShowButtons();
+    displayFrameTotals();
+  });
+
+  function createScoreCardAndButtonsHTML() {
+    for (var i = 0; i < 11; i++) {
+      $('.roll-buttons').append(
+        `<button class="roll-score" id="roll${i}" value="${i}">${i}</button>`
+      )}
+
+    for (var i = 9; i > 0; i--) {
+    $('.scorecard').prepend(`<div class="scorecard-box" id="f${i}">
+      <div class="header">Frame ${i}</div>
```

Comment:
I like that you've kept this DRY. For next level OO, you could try combining this with your displayFrames method below and have a frameView class. It might look something like:

```
function FrameView(parentElement) { 
  this.parentElement;
}

FrameView.prototype.render = function() {
   .. a variation of your code from createScoreCardAndButtonsHTML, appended to parentElement ...
}

FrameView.prototype.update = function() {
  ... a variation of your code from displayFrames...
}
```